types/User.js

```diff
@@ -113,6 +117,12 @@ const UserRegistration = S.object()
     S.string().maxLength(8)
   )
 
+const UpdateUsername = S.object()
+  .prop(
+    'username',
+    S.string().maxLength(64).pattern("^[a-zA-Z0-9-]+$").required()
```

Comment:
We should have the definition of a username pulled out and defined somewhere in this file, and then used here and on L72. 

Also it might be simpler to use `\w` as the pattern: "A word character is a character from a-z, A-Z, 0-9, including the _ (underscore) character."

So `.pattern("^\w+$")`. I'm not sure if we want to care about "minimum length" for usernames; as the pattern stands, `0` is a valid Flossbank username. I've seen in some places they enforce min length of 3.

Reply:
done